pact/coin-contract/coin.repl

```diff
@@ -1,12 +1,13 @@
 (begin-tx)
 
+(load "fungible-v1.pact")
 (load "coin.pact")
 
 (commit-tx)
 (begin-tx)
 
-;; HUGE GLARING TODO - pending FV fixes in Pact
-;(verify 'coin)
+; HUGE GLARING TODO - pending FV fixes in Pact
```

Comment:
`verify` is uncommented, do we need this comment

Reply:
no - it's just warnings atm